packages/router/__tests__/router-test.ts

```diff
@@ -362,6 +364,9 @@ function setup({
       return enhancedRoute;
     });
   }
+  if (ref) {
+    ref.enhanceRoutes = enhanceRoutes;
```

Comment:
If we go this `enhanceRoutes` approach I would just add it to the object returned from `setup` and avoid the need for the `ref`?

Reply:
removed ref, return enhanceRoutes, add helper for _internalSetRoutes